app/components/forms/wizard/basic-details-step.js

```diff
@@ -128,6 +128,15 @@ export default Component.extend(FormMixin, EventWizardMixin, {
 
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
+    window.$.fn.form.settings.rules.checkMaxMinPrice = () => {
+      return this.$('.ui.form').form('get value', 'min_price') <= this.$('.ui.form').form('get value', 'max_price');
+    };
+    window.$.fn.form.settings.rules.checkMaxMinOrder = () => {
+      return this.$('.ui.form').form('get value', 'ticket_min_order') <= this.$('.ui.form').form('get value', 'ticket_max_order');
+    };
+    window.$.fn.form.settings.rules.checkMaxTotal = () => {
```

Comment:
Check if server validations are present for them or not.
 If not then first server validations need to be added then the FE

Reply:
@kushthedude Yes the server validation are already added.

Reply:
![Screenshot from 2020-01-29 15-56-54](https://user-images.githubusercontent.com/46647141/73348823-1dcef080-42b0-11ea-9ca9-279f41ec1ee3.png)

![Screenshot from 2020-01-29 15-57-14](https://user-images.githubusercontent.com/46647141/73348831-21627780-42b0-11ea-8496-017586a6f409.png)